packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -387,3 +396,9 @@ export interface AccountHistoryCountOptions {
   txtype?: DfTxType
   no_rewards?: boolean
 }
+
+export interface CommunityBalanceData {
+  AnchorReward: BigNumber
+  IncentiveFunding?: BigNumber
+  Burnt: BigNumber
```

Comment:
Need handle this props as well

```suggestion
  AnchorReward?: BigNumber
  IncentiveFunding?: BigNumber
  Swap?: BigNumber
  Futures?: BigNumber
  Options?: BigNumber
  Unallocated?: BigNumber
  Unknown?: BigNumber
  Burnt: BigNumber
```
see https://github.com/DeFiCh/ain/blob/8e6cdc7a4a681066c5959ffbea2bc065722f1857/src/masternodes/incentivefunding.h#L25-L32

Reply:
Thank you @canonbrother. Amended! I set the `expected: false` , the test failed when it was set to `true`. Is it because these props are not included in the output when you enter `listcommunitybalances` via app's CLI?

Reply:
Those **Swap, Futures, Options, Unallocated & Unknown** are skipped due to unused balance will be burnt (as comment noticed on the code)
https://github.com/DeFiCh/ain/blob/8e6cdc7a4a681066c5959ffbea2bc065722f1857/src/masternodes/rpc_accounts.cpp#L1481-L1498
but my thought was they are part of community balances, so i requested to add the props, just in case, if they are not skipped in the future, we don't need update here again.
And in code perspective, the interface `CommunityBalanceData` is widely used.